### PR TITLE
update register

### PR DESCRIPTION
**Pull request by soumanpaul**

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ const mongoose = require('mongoose');
 const cors = require('cors');
 const User = require('./models/User');
 const Appointment = require('./models/Appointment');
-const bcrypt = require('bcrypt');
+// const bcrypt = require('bcrypt');
+const bcrypt = require('bcryptjs');
```
Comment (yashksaini-coder) [inline backend/server.js:7]: Overlooked this, nice one

File: src/pages/Login.tsx [hunk 2]
```diff
@@ -13,7 +13,8 @@ export default function Login() {
     setIsLoading(true);
 
     // Simulate API call
-    await new Promise((resolve) => setTimeout(resolve, 1000));
+    // await new Promise((resolve) => setTimeout(resolve, 1000));
```
Comment (yashksaini-coder) [inline src/pages/Login.tsx:16]: Login api to be added here with POST, setup

File: src/pages/Register.tsx [hunk 3]
```diff
@@ -4,18 +4,38 @@ import { UserPlus } from 'lucide-react';
...
+    // const responce = fetch()
+    // 
+    const data = {
+      firstName
+
+    }
+    const config = {
+      
+    }
+    axios.post(`https://medcaresystem-5sag--5173--d3acb9e1.local-credentialless.webcontainer.io/register`, data, config)
```
Comment (yashksaini-coder) [inline src/pages/Register.tsx:29]: Has to be changed in the local deployed server setup, basically the backend url for mongodb running

**Review by yashksaini-coder — APPROVED** [review]
All changes are fine, new mistakes and minor fixes are also pointed out.

File: src/pages/Login.tsx (hunk 2 above)
Comment (Sakshidiwedi-28) [reply src/pages/Login.tsx:16]: Indeed, there will be issues with the appointment section.

**Review by Sakshidiwedi-28 — COMMENTED** [review]
This code has been updated to fix the issue that was missing.

**Pull request merged**